modules/azure/logic_app_standard/variables.tf

```diff
@@ -34,10 +34,13 @@ variable "enabled" {
   default     = true
 }
 
-variable "use_managed_identity" {
-  type        = bool
-  description = "Use Managed Identity for this logic app"
-  default     = false
+variable "identity" {
+  type = object({
+    identity_type = string, # use one of these values: "SystemAssigned", "UserAssigned", "SystemAssigned, UserAssigned"
+    identity_ids  = optional(list(string), [])
```

Comment:
Have you checked is the default of `[]` is the same as not adding an identity_ids at all?

Reply:
Yes, terraform provider sets empty array inside state if identity_ids is null.